src/azure-cli-core/azure/cli/core/_config.py

```diff
@@ -22,7 +22,7 @@ class AzConfig(object):
                        '0': False, 'no': False, 'false': False, 'off': False}
 
     def __init__(self):
-        self.config_parser = configparser.SafeConfigParser()
+        self.config_parser = configparser.ConfigParser()
```

Comment:
The change is to avoid this spam:
```Python
SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  global_config = configparser.SafeConfigParser()
c:\users\trpresco\documents\github\azure-cli\src\azure-cli-core\azure\cli\core\cloud.py:209: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  config = configparser.SafeConfigParser()
c:\users\trpresco\documents\github\azure-cli\src\azure-cli-core\azure\cli\core\cloud.py:222: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  config = configparser.SafeConfigParser()
```